src/Eagle.ts

```diff
@@ -84,6 +84,7 @@ export class Eagle {
 
     static nodeDropped : Element;
     static nodeDropLocation = {x:0, y:0}; // if this remains x=0,y=0, the button has been pressed and the getNodePosition function will be used to determine a location on the canvas. if not x:0, y:0, it has been over written by the nodeDrop function as the node has been dragged into the canvas. The node will then be placed into the canvas using these co-ordinates.
+    static nodeInspectorCount : number = 0; // this is 0 if all menus collapsed and counts how many are open
```

Comment:
I think it is important to know when the inspectorCount is zero, and when it is non-zero. Those are the two cases in which the code needs to behave differently. So I ended up changing this to a boolean.